src/actions/__mocks__/selectors.js

```diff
@@ -0,0 +1 @@
+export const getNodes = nodes => nodes;
```

Comment:
You don't need this mock.
As I said in the issue:
> You don't need create a test for these actions: `removeNode`, `addParent`, `removeParent`, `changeNodeStates`. I will change them and then I will create the tests.

So, you can remove this mock and the tests for theses action.

Reply:
I'll fix it